README.md

```diff
@@ -1,63 +1,14 @@
 # WebView2 Samples
 
-This repository contains getting started apps as well as sample apps that demonstrate the features and usage patterns of [WebView2](https://docs.microsoft.com/microsoft-edge/webview2/). As we add more features to WebView2, we will regularly update our samples.
+Welcome to the WebView2Samples repo.  This repo contains several types of samples for [WebView2](https://docs.microsoft.com/microsoft-edge/webview2/):
 
-In the [GettingStarted](./GettingStartedGuides/) folder you will find the starter code for its respective guide listed below:
-- [Win32 Getting Started](https://docs.microsoft.com/microsoft-edge/webview2/gettingstarted/win32)
-- [WPF Getting Started](https://docs.microsoft.com/microsoft-edge/webview2/gettingstarted/wpf)
-- [WinForms Getting Started](https://docs.microsoft.com/microsoft-edge/webview2/gettingstarted/winforms)
-- [WinUI Getting Started](https://docs.microsoft.com/microsoft-edge/webview2/gettingstarted/winui)
+*  Getting Started tutorial projects - Completed Visual Studio projects that result from following the steps in the [Getting Started tutorials](https://docs.microsoft.com/microsoft-edge/webview2/get-started/get-started).
 
-In the [SampleApps](./SampleApps/) folder you will find:
-- [WebView2Samples.sln](SampleApps/WebView2Samples.sln) - a collective solution that includes [WebView2APISample.vcxproj](SampleApps/WebView2APISample/WebView2APISample.vcxproj), [WebView2SampleWinComp.vcxproj](SampleApps/WebView2SampleWinComp/WebView2SampleWinComp.vcxproj), [WebView2WpfBrowser.csproj](SampleApps/WebView2WpfBrowser/WebView2WpfBrowser.csproj), [WebView2WindowsFormsBrowser.csproj](SampleApps/WebView2WindowsFormsBrowser/WebView2WindowsFormsBrowser.csproj), [WV2DeploymentWiXCustomActionSample](/SampleApps/WV2DeploymentWiXCustomActionSample/README.md), and [WV2DeploymentWiXBurnBundleSample](/SampleApps/WV2DeploymentWiXBurnBundleSample/README.md).
+*  Sample apps - WebView2 sample apps for various frameworks and platforms, as Visual Studio projects.  For more information, see [Sample apps](https://docs.microsoft.com/microsoft-edge/webview2/code-samples-links).
 
-Share feedback in the [WebView2 Feedback repo](https://aka.ms/webviewfeedback).
+*  Deployment samples - Samples that demonstrate deploying the WebView2 Runtime.  For more information, see [Deployment samples](https://docs.microsoft.com/microsoft-edge/webview2/samples/deployment-samples).
 
-## Win32 C/C++
 
-#### Getting Started
+#### Open Source repo
```

Comment:
maybe add to top level of repo a file like
https://github.com/MicrosoftDocs/edge-developer/blob/main/CODE_OF_CONDUCT.md

Reply:
exists.
https://github.com/MicrosoftEdge/WebView2Samples/blob/main/.github/CODE_OF_CONDUCT.md